pytest_splinter/plugin.py

```diff
@@ -268,22 +307,9 @@ def browser_instance_getter(
     :return: function(parent). Each time this function will return new instance of plugin.Browser class.
     """
     def get_browser():
-        kwargs = {}
-
-        if splinter_webdriver == 'firefox':
-            kwargs['profile_preferences'] = dict({
-                'browser.download.folderList': 2,
-                'browser.download.manager.showWhenStarting': False,
-                'browser.download.dir': splinter_file_download_dir,
-                'browser.helperApps.neverAsk.saveToDisk': splinter_download_file_types,
-                'browser.helperApps.alwaysAsk.force': False,
-                'pdfjs.disabled': True,  # disable internal ff pdf viewer to allow auto pdf download
-            }, **splinter_firefox_profile_preferences)
-            kwargs['profile'] = splinter_firefox_profile_directory
-        elif splinter_webdriver == 'remote':
-            kwargs['url'] = splinter_remote_url
-        if splinter_driver_kwargs:
-            kwargs.update(splinter_driver_kwargs)
+        kwargs = get_args(splinter_webdriver, splinter_file_download_dir, splinter_download_file_types,
```

Comment:
Don't follow your question.

get_args returns a dict which is passed as kwargs to the Browser() constructor. get_args has keyword arguments.